runtime/pom.xml

```diff
@@ -44,36 +44,43 @@
     <finalName>runtime</finalName>
     <plugins>
       <plugin>
-        <groupId>org.wildfly.swarm</groupId>
-        <artifactId>wildfly-swarm-plugin</artifactId>
-        <version>${wildfly.swarm.version}</version>
-        <configuration>
-          <mainClass>com.redhat.ipaas.runtime.Main</mainClass>
-          <jvmArguments>
-            <jvmArgument>-server</jvmArgument>
-            <jvmArgument>-Xms256m</jvmArgument>
-            <jvmArgument>-Xmx512m</jvmArgument>
-          </jvmArguments>
```

Comment:
So where did this figure come from? Can you please share your calculations on how this amount of memory is reasonable for our REST service runtime? These are just arbitrary numbers so for now we should remove and refine it once we have some NFRs and some actual runtime numbers.